### ISSUE TITLE
Serious balance issues in early game

### ISSUE THREAD

**Issue by afwbkbc** [freeorion/freeorion#853]
Beginning of the game has very limited number of choices available - basically it's just choosing which lane to scout and which planet to colonise. Sonetimes galaxy is generated in such a way that there are 0 habitable planets and zero natives around capital. And finally, when you found at least something, it is already occupied by another empire. After that, all you can do is build military and try to capture it, but empire owning it is typically stronger or at least matches your power. So, what can you do? The only choice is to sit on one planet and gather forces / research military techs. During this time you will get zero industry and research growth, while other players are expanding and advancing science. Time will be really against you - if you don't capture this planet fast, you will become far behind in military and will never capture it. And even if you capture it, you will still be behind enough to lose the game in the end. And as scouting is very costly in early game, you will have to send fleet blindly, and if you're unlucky it will be destroyed and you will need another 20-30 turns to rebuild it (but then it will already be useless). And, even worse, you will have to send troop ships and outpost ships there, but they will be defenceless even if there's your fleet in destination system. In last game, after I destroyed enemy fleet and planet defences I sent outpost ship to colonize another pkanet in same system. It seemed safe as there was my fleet stationed there. But when outpost ship was in lane to this planet and had 1 turn to destination, enemy ship appeared on other lane, with one turn left too. So, guess what? He destroyed my outpost ship before was destroyed by my fleet. To me it meant anothet 6 turns to build another outpost ship and another 5 turns to get to this pkanet again. Meanwhile, enemies grow and expand. So I surrendered. I couldn't build scouts because they would get destroyed easily by enemy. This pkanet had to be captured urgently, no time to waste 4 turns on every scout.

So, to sum up: there is a random chance that you will lose the game without being able to do anything. This is serious issue in competitive multiplayer games.
Solution would be adding algorithm that checks generated galaxy to see if there are such unbalanced conditions in early game for any player, and tgen fix them by adding some advantages like pkanets or removing some monsters. Regardless of galaxy choices there must not be such disadvantage in early game for any player.

p.s. also Chato must be nerfed, but that's another story :)

kthx

**Comment by Vezzra** [comment]
@afwbkbc, your thoughts and input is certainly appreciated, you raise points worthy of discussion, but why do you persist in using our issue tracker to share them? Like the last three issues you opened, the proper place to discuss the things you raise here is our [forum](http://www.freeorion.org/forum/).

And I want to say it once again, you are very welcome to share all your thoughts and ideas there. I don't know what else to say, as you haven't responded to my other appeals to head over to our forum at all.

I didn't want to just close all the issues you opened, because I don't want you to get the impression that we don't care about what you have to say. We do care, just not here on the issue tracker, at least not for the topics you've risen.

**Comment by Vezzra** [comment]
Closing this as "invalid". Please post this thoughts on the forum as suggested.
